### PR TITLE
Tech 298: DS spacing

### PR DESCRIPTION
**Pull request by thaisribon**
# Description

- DS spacing
- I had to disregard the % and the negative values(these are the exceptions)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Dangerous changes
- [x] Chore (change that does not effect how the code works, eg: change color)
- [ ] Tests

## How to test

Put here all the things need to test this PR and verify your changes, also list any relevant details for tests (eg: have a wallet)

- Test A

### CODE REVIEW

File: src/components/atomics/buttons/Button/styles.ts [hunk 1]
```diff
@@ -54,15 +54,15 @@ export const Container = styled.button<{
...
   }
 
   #right-icon {
-    margin-left: 4px;
+    margin-left: ${({ theme }) => theme.spacing(4)};
   }
 
   svg {
-    margin-left: 2px;
+    margin-left: ${({ theme }) => theme.spacing(4)};
```
Comment (letbueno) [inline src/components/atomics/buttons/Button/styles.ts:65]: só pra ter certeza, esse tam mudou no handoff?

Comment (thaisribon) [reply src/components/atomics/buttons/Button/styles.ts:65]: yep, we don't have 2 anymore

File: src/styles/normalize.ts [hunk 2]
```diff
@@ -122,15 +122,15 @@ const normalize = css`
   }
 
   fieldset {
-    padding: 0.35em 0.75em 0.625em;
+    padding: ${({ theme }) => theme.spacing(4, 12, 12)};
   }
 
   legend {
     box-sizing: border-box; /* 1 */
     color: inherit; /* 2 */
```
Comment (letbueno) [inline src/styles/normalize.ts:130]: why we have this comments? i don't understand them kkk

File: src/styles/theme.ts [hunk 3]
```diff
@@ -14,6 +14,15 @@ interface ThemeType {
   [key: string]: any;
 }
 
+const getSpacing = (space: number): string => {
+  const spacings = [0, 4, 8, 12, 16, 20, 24, 32, 40, 48, 64, 80, 112];
```
Comment (letbueno) [inline src/styles/theme.ts:18]: i love this solution!!

**Review by letbueno — APPROVED** [review]

File: src/styles/normalize.ts (hunk 2 above)
Comment (thaisribon) [reply src/styles/normalize.ts:130]: i don't know :/

**Pull request merged**